### PR TITLE
Code forces 266 b

### PR DESCRIPTION
**Pull request by izaLukawska**

### CODE REVIEW

File: src/CodeForces/CodeForces_266B.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+//https://codeforces.com/problemset/problem/266/B
+
+public class CodeForces_266B {
+
+    public static String queue(int kids, int seconds, String sex) {
+        if (kids < 1 && kids > seconds) {
+            throw new IllegalArgumentException("Kids number must be bigger than 1 and less than seconds count");
+        } else if (seconds > 50 && seconds < kids) {
+            throw new IllegalArgumentException("Seconds count must be bigger than kids number but less than 50");
+        } else {
```
Comment (Wlodek-YouCanDev) [inline src/CodeForces/CodeForces_266B.java:16]: Why we need else block here if we throws exceptions?

File: src/CodeForces/CodeForces_266B.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package CodeForces;
+
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.stream.Collectors;
+
+//https://codeforces.com/problemset/problem/266/B
+
+public class CodeForces_266B {
```
Comment (Wlodek-YouCanDev) [inline src/CodeForces/CodeForces_266B.java:9]: There is no possibility to run this code and to check that it works correctly

File: src/CodeForces/CodeForces_266B.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+    public static String queue(int kids, int seconds, String sex) {
+        if (kids < 1 && kids > seconds) {
+            throw new IllegalArgumentException("Kids number must be bigger than 1 and less than seconds count");
+        } else if (seconds > 50 && seconds < kids) {
+            throw new IllegalArgumentException("Seconds count must be bigger than kids number but less than 50");
+        } else {
+            LinkedList<String> list = Arrays.stream(sex.split("")).collect(Collectors.toCollection(LinkedList::new));
```
Comment (Wlodek-YouCanDev) [inline src/CodeForces/CodeForces_266B.java:17]: Why here is a LinkedList type? Maybe it is better to use List type to make it convertable?

File: src/CodeForces/CodeForces_266B.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+            for(int j = 0; j < seconds; ++j) {
+                for(int i = list.size() - 1; i > 0; --i) {
+                    if (((String)list.get(i)).equals("G") && ((String)list.get(i - 1)).equals("B")) {
```
Comment (Wlodek-YouCanDev) [inline src/CodeForces/CodeForces_266B.java:21]: Why you convert object from list to String? Do we need (String) here?

**Review by Wlodek-YouCanDev — CHANGES_REQUESTED** [review]

**Comment by izaLukawska** [comment]
ready for review

**Review by Wlodek-YouCanDev — APPROVED** [review]

**Pull request merged**